CavalcanteLucas/labdale
# Feature/OL211-delete_todo

**Link to task:**
https://labcodes.atlassian.net/browse/OL-211

**How to test:**
With a user logged in, create a TodoList, then create a Todo. After that, try to delete the Todo with the small bin besides the Todo title. Check if modal for confirmation pops up. Confirm delete and check if Todo is removed from the list of Todos. See if a dismissable notification of success appear on the DashboardPage when operation is complete.

**Description of your solution:**
This PR implements the backend API and the frontend interfaces that enable Todos to be individually deleted. When deleting a Todo, a modal for confirmation is open. After confirmation, that instance of todo list is deleted. Notification of success should appear on the topscreen of the dashboard. New tests introduced by this PR covers DELETE requests to Todos.

**Observations**
This PR should be merged on master branch after merge of [PR16](https://github.com/CavalcanteLucas/labdale/pull/16)
